norddrop/ffi/norddrop.i

```diff
@@ -84,12 +84,12 @@ struct norddrop {};
 
     // Purge all transfers that are older than the given timestamp from the
     // database. Accepts a UNIX timestamp in seconds
-    enum norddrop_result purge_transfers_until(int64_t until_timestamp);
+    enum norddrop_result purge_transfers_until(unsigned long long until_timestamp);
```

Comment:
Are you sure that `unsigned long long` is exactly 8 bytes on all supported platforms?

Reply:
Rust doesn't have a say in SWIG and already uses properly sized variable. I just guessed that `ull` would work and it seems to produce proper binding code.

I would say it's pretty hard to decide if the generated bindings are OK exactly, that's for apps to say, so that's my approach